Common/Core/TrackSelectionDefaults.cxx

```diff
@@ -107,4 +107,24 @@ TrackSelection getGlobalTrackSelectionRun3HF()
   return selectedTracks;
 }
 
+// Reduced default track selection for jet validation based on hybrid cuts for converted (based on ESD's from run 2) A02D's
+TrackSelection getJEGlobalTrackSelectionRun2()
+{
+  TrackSelection selectedTracks;
+  selectedTracks.SetTrackType(o2::aod::track::Run2Track); // Run 2 track asked by default
+  selectedTracks.SetMinNCrossedRowsTPC(70);
+  selectedTracks.SetMinNCrossedRowsOverFindableClustersTPC(0.8f);
+  selectedTracks.SetMaxChi2PerClusterTPC(4.f);
+  selectedTracks.SetRequireTPCRefit(true);
```

Comment:
Here I would still use the main method and highlight what are the differences, i.e. you can start with   `TrackSelection selectedTracks = getGlobalTrackSelection();` and then build on that, we should do the same for the hf cut btw